webui/react/src/hooks/useFeature.ts

```diff
@@ -0,0 +1,26 @@
+import queryString from 'query-string';
+
+import { useStore } from 'contexts/Store';
+
+type ValidFeature = 'rbac' // Add new feature switches here using `|`
+const queryParams = queryString.parse(window.location.search);
+
+interface FeatureHook {
+  isOn: (feature: ValidFeature) => boolean
+}
+
+const useFeature = (): FeatureHook => {
+  return { isOn: (ValidFeature) => IsOn(ValidFeature) };
+};
+
+const IsOn = (feature: string): boolean => {
+  const { info: { rbacEnabled } } = useStore();
```

Comment:
You should move this into `useFeature` and inline the rest of `IsOn` otherwise you could potentially get hooks errors when using this.

Reply:
Oh I see, so these are pages that are now only linked to if rbac is on? In That case this looks good to me.